common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java

```diff
@@ -58,9 +59,11 @@ public class MLRegisterModelInput implements ToXContentObject, Writeable {
     public static final String BACKEND_ROLES_FIELD = "backend_roles";
     public static final String ADD_ALL_BACKEND_ROLES_FIELD = "add_all_backend_roles";
     public static final String DOES_VERSION_CREATE_MODEL_GROUP = "does_version_create_model_group";
+    public static final String GUARDRAILS_FIELD = "guardrails";
 
     private static final Version MINIMAL_SUPPORTED_VERSION_FOR_DOES_VERSION_CREATE_MODEL_GROUP = Version.V_2_11_0;
     private static final Version MINIMAL_SUPPORTED_VERSION_FOR_AGENT_FRAMEWORK = Version.V_2_12_0;
+    private static final Version MINIMAL_SUPPORTED_VERSION_FOR_GUARDRAILS = Version.V_2_13_0;
```

Comment:
Good to add this version checking here. But why don't add similar checking for other classes like MLUpdateModelInput?

Reply:
as above

Reply:
For safe, suggest add version check for all places